addons/website_mass_mailing/static/src/snippets/s_newsletter_block/options.js

```diff
@@ -31,24 +58,32 @@ options.registry.mailing_list_subscribe = options.Class.extend({
                     });
                 },
                 cancel: () => {
-                    this.trigger_up("remove_snippet", {
+                    this.website.websiteRootInstance.trigger_up("remove_snippet", {
```

Comment:
Same here, I do not think this works. You should probably add removeSnippet as a callback passed from the SnippetEditor